kubernetes/replicator-gke-cc/docs/index.rst

```diff
@@ -0,0 +1,198 @@
+
+Google Kubernetes Engine to |ccloud| with |crep-full|
+=====================================================
+
+Overview
+--------
+
+This demonstrations features a deployment of |cp| on Google Kubernetes Engine (GKE) leveraging |co-long| and |crep-full|, highlighting a data replication strategy to |ccloud|.  Upon running this demo, you will have a GKE based |cp| deployment with simulated data replicating to your |ccloud| cluster.  We will verify the replication by running client applications against the |ccloud| cluster to view the simulated data originating in the source GKE cluster.
+
+The major components of this demo are:
+
+* A |ccloud| environment
+* A Kubernetes cluster running on GKE.
+* |co-long| which is used to manage the following |cp| components
+
+  * A 3 node |zk| cluster
+  * A 3 node node |ak| cluster
+  * A single node |crep|
+  * A single node |sr|
+  * A single node |kconnect-long|
+  * |c3|
+  * One instance of ``kafka-connect-datagen`` to produce mock stock trading data
+
+.. figure:: images/operator-demo-phase-2.png
+    :alt: operator
+
+Demo Prerequisites
+-------------------
+The following applications or libraries are required to be installed and available in the system path in order to properly run the demo.
+
++------------------+----------------+----------------------------------------------------------+
+| Application      | Tested Version | Info                                                     |
++==================+================+==========================================================+
+| ``kubectl``      | ``1.14.3``     | https://kubernetes.io/docs/tasks/tools/install-kubectl/  |
++------------------+----------------+----------------------------------------------------------+
+| ``helm``         | ``2.14.3``     | https://helm.sh/docs/using_helm/#install-helm            |
++------------------+----------------+----------------------------------------------------------+
+| ``gcloud``       | ``267.0.0``    |  https://cloud.google.com/sdk/install                    |
+| ``GCP sdk core`` | ``2019.10.15`` |                                                          |
++------------------+----------------+----------------------------------------------------------+
+| ``ccloud``       | ``v0.185.0``   | https://docs.confluent.io/current/cloud/cli/install.html |
++------------------+----------------+----------------------------------------------------------+
+
+Running the Demo
+----------------
+
+.. warning:: This demo uses the real GCP SDK to launch real resources. To avoid unexpected charges, carefully evaluate the cost of resources before launching the demo and ensure all resources are destroyed after you are done evaluating the demonstration.  Optionally, refer to the |co| :ref:`Sizing Recommendations <co-env-sizing>` document and the :ref:`examples-operator-gke-base-variable-reference` section for more information on required resources for running |cp| on Kubernetes.
```

Comment:
Consider adding to the docs explanation on how to destroy resources.  Perhaps I'm missing it, but I don't see that on this page.

Reply:
WIP